src/views/LoginView/LoginView.js

```diff
@@ -1,25 +1,18 @@
-import styles from './LoginView.scss'
-import autobind from 'autobind-decorator'
-import cx from 'classnames'
 import React, { Component, PropTypes } from 'react'
-import { Button, Field, Form, Header, Input, Message } from 'stardust'
 import { connect } from 'react-redux'
-import { bindActionCreators } from 'redux'
-import { login } from 'services/session'
+import {
+  Grid,
+  Header,
+  Image,
+  Column,
+} from 'stardust'
 
-const mapStateToProps = (state) => ({
-  session: state.session,
-})
-
-const mapDispatchToProps = (dispatch) => ({
-  actions: bindActionCreators({
-    login,
-  }, dispatch),
-})
+import './LoginView.scss'
+import LoginForm from 'components/LoginForm/LoginForm'
+import logoUrl from 'assets/images/android-icon-192x192.png'
```

Comment:
what if we switch CDNs or something, should have static URL variable instead of `assets/` literal?